src/utils/randomNum.js

```diff
@@ -0,0 +1,17 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+
+export const randomNum = () => {
+	const arr = []
+
+	while (arr.length < 3) {
+		const randomNumber = MissionUtils.Random.pickNumberInRange(1, 9)
+
+		let overlap = arr.indexOf(String(randomNumber))
+
+		if (overlap === -1) {
+			arr.push(String(randomNumber))
+		}
```

Comment:
```suggestion
if (arr.includes(randomNumber)) {
    arr.push(randomNumber)
}
```
index로 중복을 체크하는 접근 방식도 좋은 것 같은데 간단하게 `includes`를 사용하거나 `some`을 사용하면 코드가 간결해질 수 있을 것 같아요!

Reply:
맞습니다! 저도 직관적인거는 includes가 더 좋다고 생각을해서 변경해야겠군요!